templates/index.html

```diff
@@ -5,7 +5,7 @@
 {{ this.title }}
 {% endblock %}
 {% block header %}
-<h1>Youngstown Action Center</h1>
+<h1><img src="yac_logo.png" width="50%"><br>Youngstown Action Center</br></h1>
```

Comment:
Add "/" to make img link absolute instead of relative

when on home page URL is:
127.0.0.1/

so image lookup goes:

127.0.0.1/yac_logo.png which works

on contact page URL is

127.0.0.1/contact/

so image lookup goes:

127.0.0.1/contact/yac_logo.png which fails

Reply:
also, `<br>` is a standalone tag, try and be consistent with having a / or no /, as written there are two separate breaks

`<hr>` is the same way, you do not need to open and close